main.py

```diff
@@ -420,18 +464,18 @@ def parse_prompt(record: dict, article: Article):
     prompt_text = record.get("fields").get(f"Prompt {article.language}", "")
     placeholders = re.findall(r'\[(.*?)]', prompt_text)
     for placeholder in placeholders:
-        #value_to_replace = record.get("fields").get(placeholder, f"No value for {placeholder}")
+        # value_to_replace = record.get("fields").get(placeholder, f"No value for {placeholder}")
         prompt_text = prompt_text.replace(f"[{placeholder}]", article.job_name)
 
     return {
-            "response": "",
-            "section": record.get("fields").get("Section Name"),
-            "plain_text": "",
```

Comment:
this are not necessary changes, but ok, let's try to avoid this in future unless it improve readability or something like that

Reply:
sure